pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -47,6 +47,7 @@ const (
 	filter                                    = "Filter"
 	postFilter                                = "PostFilter"
 	score                                     = "Score"
+	scoreExtensionNormalize                   = "ScoreExtensionNormalize"
```

Comment:
This looks like it's only used in one spot (https://github.com/kubernetes/kubernetes/pull/84522/files#diff-76b7e314f9618c8bc2ca6442ddd35dc3R448), does that need to be a constant?

Reply:
I don't think whether a var is used once or multiple times determines whether it should be a constant or not. Having it here is just more consistent and easy to read with other constant strings. Let me know if you think otherwise.